server/code/setup/cpdb

```diff
@@ -131,6 +131,20 @@ class PostgresqlSetup(DbSetup):
             # Append / for the database name:
             self.jdbc_url = "%s/" % (self.jdbc_url)
         self.jdbc_url = "%s%s" % (self.jdbc_url, db)
+
+        # Appending SSL connection details
+        if ssl == True:
+            if sslfactory is None:
+                if sslcertpath is None:
+                    self.jdbc_url = "%s%s" % (self.jdbc_url, "?ssl=true&sslfactory=org.postgresql.ssl.NonValidatingFactory")
+                    print("Default sslfactory=org.postgresql.ssl.NonValidatingFactory is used as you have not provided factory details.")
+                elif sslcertpath is not None:
+                    self.jdbc_url = "%s%s%s" % (self.jdbc_url, "?ssl=true&sslmode=verify-full&sslrootcert=", sslcertpath)
+
+            elif sslfactory is not None:
+                if sslcertpath is None:
+                    self.jdbc_url = "%s%s%s" % (self.jdbc_url, "?ssl=true&sslfactory=",sslfactory)
```

Comment:
One last whitespace update here: need a space between the last two arguments here.

Reply:
Thanks for noticing that. I corrected it now.